diffsims/sims/kinematic_simulation.py

```diff
@@ -0,0 +1,187 @@
+"""
+Created on 1 Nov 2019
+
+Back end for computing diffraction patterns with a kinematic model.
+
+@author: Rob Tovey
+"""
```

Comment:
We need to put the standard copyright notice at the top of all files in this PR.

Reply:
I'll do this too